packages/nc-gui/components/cell/MultiSelect.vue

```diff
@@ -336,6 +337,7 @@ useEventListener(document, 'click', handleClose, true)
         :key="op.id || op.title"
         :value="op.title"
         :data-testid="`select-option-${column.title}-${rowIndex}`"
+        :class="`nc-select-option-${column.title}-${op.title}`"
```

Comment:
You can use `getByTestId` or `[data-testid="val"]` in playwright to select and class name can be removed

Reply:
@dstala